jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/execution/sam/SamCommon.kt

```diff
@@ -25,8 +25,11 @@ class SamCommon {
         val SAM_BUILD_DIR = ".aws-sam"
         val SAM_INFO_VERSION_KEY = "version"
 
+        // Inclusive
         val expectedSamMinVersion = SemVer("0.7.0", 0, 7, 0)
-        val expectedSamMaxVersion = SemVer("0.8.0", 0, 8, 0)
+
+        // Exclusive
+        val expectedSamMaxVersion = SemVer("0.110", 0, 11, 0)
```

Comment:
missing a dot - `0.11.0` and not `0.110`

Reply:
Andd thats why we have tests